src/network/networkd-network-gperf.gperf

```diff
@@ -110,6 +110,9 @@ Bridge.AllowPortToBeRoot,               config_parse_bool,
 Bridge.UnicastFlood,                    config_parse_bool,                              0,                             offsetof(Network, unicast_flood)
 BridgeFDB.MACAddress,                   config_parse_fdb_hwaddr,                        0,                             0
 BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,                       0,                             0
+BridgeVLAN.PVID,                        config_parse_uint32,                            0,                             offsetof(Network, pvid)
```

Comment:
hmm, so it appears the rest of the vlan code actually is totally chaotic too, and src/network/networkd-netdev-vlan.h even uses uint64_t for it... yuck... this really needs some clean-up i think... (i understand though if you don't want to to the clean-up in your patchset)